src/render/mod.rs

```diff
@@ -21,7 +21,8 @@ pub async fn render_stream(
     if stdout().is_terminal() {
         let render_options = config.read().get_render_options()?;
         let mut render = MarkdownRender::init(render_options)?;
-        markdown_stream(rx, &mut render, &abort).await
+        let model_id = config.read().model_id.clone();
+        markdown_stream(rx, &mut render, &abort, &model_id).await
```

Comment:
I put `config.read().model_id.clone()` in a temporary variable intentionally in L24.
Because when I ran `cargo build` without temporary variable like `markdown_stream(rx, &mut render, &abort, &config.read().model_id.clone()).await`, it provided error message like this:
```sh
error: future cannot be sent between threads safely
   --> src/main.rs:176:1
    |
176 | #[async_recursion::async_recursion]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ future created by async block is not `Send`
    |
    = help: within `{async block@src/main.rs:176:1: 176:36}`, the trait `Send` is not implemented for `*mut ()`, which is required by `{async block@src/main.rs:176:1: 176:36}: Send`
note: future is not `Send` as this value is used across an await
   --> src/render/mod.rs:25:83
    |
25  |         markdown_stream(rx, &mut render, &abort, &config.read().model_id.clone()).await
    |                                                   -------------                   ^^^^^ await occurs here, with `config.read()` maybe used later
    |                                                   |
    |                                                   has type `parking_lot::lock_api::RwLockReadGuard<'_, parking_lot::RawRwLock, config::Config>` which is not `Send`
    = note: required for the cast from `Pin<Box<{async block@src/main.rs:176:1: 176:36}>>` to `Pin<Box<dyn futures_util::Future<Output = Result<(), anyhow::Error>> + Send>>`
    = note: this error originates in the attribute macro `async_recursion::async_recursion` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `aichat` (bin "aichat") due to 1 previous error
```